.travis.yml

```diff
@@ -0,0 +1,21 @@
+# vim: set tabstop=2 shiftwidth=2 expandtab:
+language: php
+
+php:
+  - 5.4
```

Comment:
You're right, I would keep only 5.4 and 5.6 and then keep only the latter after our upgrade